app/views/physical_server/show.html.haml

```diff
@@ -2,5 +2,7 @@
     - arr = %w(physical_server physical_servers)
     - if arr.include?(@display) && @showtype != "compare"
         = render(:partial => "layouts/gtl", :locals => {:action_url => "show/#{@ph_server.id}"})
+    - elsif @showtype == 'main'
+        = render :partial => "layouts/textual_groups_generic"
     - else
         = render(:partial => @showtype)
```

Comment:
please, don't do this: 
`render(:partial => @showtype)`

I guess you don't need the else branch at all. And if you do, then list all the `@showtype` values explicitly. It's hard to figure the possible values for `@showtype` later on and it is a dangerous practice. If user data would reach `@showtype` then this can lead to a security issue.

I bet you can just remove it for you case.